packages/webpack/src/presets/base.ts

```diff
@@ -125,7 +125,7 @@ function baseAlias (ctx: WebpackConfigContext) {
     ...ctx.alias,
   }
   if (ctx.isClient) {
-    ctx.alias['#internal/nitro'] = resolve(ctx.nuxt.options.buildDir, 'nitro.client.mjs')
+    ctx.alias['nitro/runtime'] = resolve(ctx.nuxt.options.buildDir, 'nitro.client.mjs')
```

Comment:
Just checking it was because of runtime config access right?

Reply:
Exactly. This is so we can use isomorphic code that gets replaced with different runtime config code on client.

Reply:
Okay! I have an idea to get rid of it!

Reply:
🙌